.appveyor.yml

```diff
@@ -15,7 +15,7 @@ install:
   - choco install vswhere
 
 build_script:
-  - bash scripts/build.sh
+  - bash scripts/build.sh --release
```

Comment:
If we build in release we'll test using the same .NET assemblies and Uno packages we ship to users, and I think this makes us safer as it will also catch problems with packaging of Uno packages.

For `uno test` projects, this flag doesn't change anything - `uno test` still makes debug builds.

Reply:
Ah, right. Yeah, I agree with that reasoning :+1: